lib/actuators/src/ContServo.cpp

```diff
@@ -1,48 +1,33 @@
 #include "ContServo.h"
 
-ContServo::ContServo(PinName pin) : m_pin(pin), m_pwm(pin) {
-  m_max_speed = 0;
-  m_speed     = 0;
-
-  m_max_pulse = DEFAULT_MAX;
-  m_min_pulse = DEFAULT_MIN;
-
-  // INIT PWM DEFAULT
-  m_pwm.period(PERIOD.count());
-}
-
-ContServo::ContServo(PinName pin, float max_speed) : ContServo(pin) {
-  m_max_speed = max_speed;
-}
-
-ContServo::ContServo(PinName pin, float max_speed, std::chrono::duration<float> max_pulse,
-                     std::chrono::duration<float> min_pulse)
-    : ContServo(pin, max_speed) {
+ContServo::ContServo::ContServo(PinName pin, float max_speed, std::chrono::duration<float> max_pulse,
+                                std::chrono::duration<float> min_pulse, std::chrono::duration<float> period)
+    : m_pwm(pin), m_max_speed(max_speed), m_speed(0) {
+  // Set initial condition of PWM
+  m_pwm.period(period.count());
+  m_pwm = 0.0;
+
+  // Set max and min pulse widths
   m_max_pulse = max_pulse;
   m_min_pulse = min_pulse;
```

Comment:
just do this in the member init list

Reply:
I can't do that cuz they're member variables of the Servo base class